### PR TITLE
Update foreman datacenter

### PR DESCRIPTION
**Pull request by ehelms**

### CODE REVIEW

File: packages/plugins/rubygem-foreman_datacenter/rubygem-foreman_datacenter.spec [hunk 1]
```diff
@@ -1,17 +1,22 @@
+# FIXME:
+#   1. Edit foreman requirement(s) and specify minimum version
+#   2. Delete these lines
+#
```
Comment (evgeni) [inline packages/plugins/rubygem-foreman_datacenter/rubygem-foreman_datacenter.spec:4]:
```suggestion
```

File: packages/plugins/rubygem-foreman_datacenter/rubygem-foreman_datacenter.spec [hunk 2]
```diff
@@ -1,17 +1,22 @@
...
+# template: foreman_plugin
 %{?scl:%scl_package rubygem-%{gem_name}}
 %{!?scl:%global pkg_name %{name}}
 %{!?_root_sysconfdir:%global _root_sysconfdir %{_sysconfdir}}
 
 %global gem_name foreman_datacenter
 %global plugin_name datacenter
-%global foreman_min_version 2.1
+%global foreman_min_version FIXME
```
Comment (evgeni) [inline packages/plugins/rubygem-foreman_datacenter/rubygem-foreman_datacenter.spec:12]:
```suggestion
%global foreman_min_version 2.1
```

File: packages/plugins/rubygem-foreman_datacenter/rubygem-foreman_datacenter.spec [hunk 3]
```diff
@@ -1,17 +1,22 @@
...
 
 Name: %{?scl_prefix}rubygem-%{gem_name}
 Version: 2.3.0
-Release: 1%{?foremandist}%{?dist}
+Release: 2%{?foremandist}%{?dist}
 Summary: A plugin that lets you document your servers in a datacenter
 Group: Applications/Systems
-License: GPLv3
+License: FIXME
```
Comment (evgeni) [inline packages/plugins/rubygem-foreman_datacenter/rubygem-foreman_datacenter.spec:19]:
```suggestion
License: GPLv3
```

File: packages/plugins/rubygem-foreman_datacenter/rubygem-foreman_datacenter.spec [hunk 4]
```diff
@@ -94,15 +99,23 @@ cp -a .%{gem_dir}/* \
...
 %{gem_instdir}/Rakefile
 %{gem_instdir}/test
 
 %changelog
+* Mon Mar 15 2021 Eric D. Helms <ericdhelms@gmail.com> - 2.3.0-2
+- Update spec to lastest template
+
+* Mon Mar 15 2021 Eric D. Helms <ericdhelms@gmail.com> 2.3.0-1
+- Update to 2.3.0-1
+
```
Comment (evgeni) [inline packages/plugins/rubygem-foreman_datacenter/rubygem-foreman_datacenter.spec:118]:
```suggestion
```

**Review by evgeni — CHANGES_REQUESTED** [review]

File: packages/plugins/rubygem-foreman_datacenter/rubygem-foreman_datacenter.spec (hunk 1 above)
Comment (ehelms) [reply packages/plugins/rubygem-foreman_datacenter/rubygem-foreman_datacenter.spec:4]: Blast, I squished the wrong commit into the wrong commit.

**Review by evgeni — APPROVED** [review]

**Pull request merged**